src/pybind/mgr/dashboard/run-backend-api-tests.sh

```diff
@@ -17,6 +17,8 @@ if [[ "$1" = "-h" || "$1" = "--help" ]]; then
 	exit 0
```

Comment:
I can't run the script on my local vstart environment anymore:

```
ceph-dev /ceph/src/pybind/mgr/dashboard # ./run-backend-api-tests.sh 
+ setup_teuthology
++ mktemp -d
+ TEMP_DIR=/tmp/tmp.VvZrbWW7xZ
++ pwd
+ CURR_DIR=/ceph/src/pybind/mgr/dashboard
+ BUILD_DIR=/ceph/src/pybind/mgr/dashboard/../../../../build
+ read -r -d '' TEUTHOLOFY_PY_REQS
```

Any ideas? - Did I do something wrong or is something missing?

Reply:
Yeah, I had a stray `-e` there - my fault, sorry. The 'debug: ...' commit may or may not be something we actually want; right now I'm trying to figure out why the ceph-dashboard-pull-requests jenkins job never passes.

Reply:
Hey @zmc there seems to be an issue with running the ``run-tox-mgr-dashboard`` tests when running make check/ceph-dashboard tests based on centos. :(  We're still investigating on that - but if you have an idea how to fix this - feedback/suggestions are much appreciated. :)